DSCResources/MSFT_xDnsARecord/MSFT_xDnsARecord.schema.mof

```diff
@@ -0,0 +1,8 @@
+﻿[ClassVersion("1.0.0.0"), FriendlyName("xDnsARecord")]
+class MSFT_xDnsARecord : OMI_BaseResource
+{
+    [Key] string Name;
+    [Required] string Zone;

```

Comment:
Seems like Zone should be part of the key, otherwise, you cannot have the same host in two zones.